src/react-apps/applications/shared/__tests__/LeftDrawerMenu.test.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import * as renderer from 'react-test-renderer';
+import LeftDrawerMenu from '../src/navigation/drawer/LeftDrawerMenu';
+
+describe('render left drawer menu', () => {
+  let mockMenuType: string;
+  let mockactiveLeftMenuSelection: string;
+
+  beforeEach(() => {
+    mockMenuType = 'create';
+    mockactiveLeftMenuSelection = 'gui';
+  });
+
+  it('renders left drawer menu for create option in header without crashing', () => {
+    const rendered =
+      renderer.create(
+        <MemoryRouter>
+          <LeftDrawerMenu menuType={mockMenuType} activeLeftMenuSelection={mockactiveLeftMenuSelection} />
```

Comment:
Suggestion: props on each line for better readability.

Reply:
Fixed